R/get.R

```diff
@@ -87,6 +87,13 @@ get <- function(value = NULL,
   # merge the specified configuration with the default configuration
   active_config <- merge_lists(default_config, do_get(config))
 
+  # check whether any expressions need to be evaluated
+  is_expr <- vapply(active_config, is.expression, logical(1))
+  if (any(is_expr)) {
```

Comment:
I think this makes the code a little harder to read, and it will have almost no performance impact. It feels like a premature optimisation to me.

Reply:
Makes sense.